frontend/catalyst/passes.py

```diff
@@ -70,68 +74,62 @@ def cancel_inverses(fn=None):  # pylint: disable=line-too-long
         dev = qml.device("lightning.qubit", wires=1)
 
         @qjit(keep_intermediate=True)
-        def workflow():
-            @cancel_inverses
-            @qml.qnode(dev)
-            def f(x: float):
-                qml.RX(x, wires=0)
-                qml.Hadamard(wires=0)
-                qml.Hadamard(wires=0)
-                return qml.expval(qml.PauliZ(0))
-
-            @qml.qnode(dev)
-            def g(x: float):
-                qml.RX(x, wires=0)
-                qml.Hadamard(wires=0)
-                qml.Hadamard(wires=0)
-                return qml.expval(qml.PauliZ(0))
-
-            ff = f(1.0)
-            gg = g(1.0)
-
-            return ff, gg
-
-    >>> workflow()
-    (Array(0.54030231, dtype=float64), Array(0.54030231, dtype=float64))
-    >>> print(get_compilation_stage(workflow, "QuantumCompilationPass"))
+        @cancel_inverses
+        @qml.qnode(dev)
+        def circuit(x: float):
+            qml.RX(x, wires=0)
+            qml.Hadamard(wires=0)
+            qml.Hadamard(wires=0)
+            return qml.expval(qml.PauliZ(0))
+
+    >>>circuit(0.54)
+    Array(0.85770868, dtype=float64)
+
+    Note that the QNode will be unchanged in Python, and will continue
+    to include self-inverse gates when inspected with Python (for example,
+    with :func:`~.draw`).
+
+    To instead view the optimized circuit, the MLIR must be viewed
+    after the ``"QuantumCompilationPass"`` stage:
+
+    >>> print(get_compilation_stage(circuit, stage="QuantumCompilationPass"))
+    module @circuit {
+      func.func public @jit_circuit(%arg0: tensor<f64>) -> tensor<f64> attributes {llvm.emit_c_interface} {
+        %0 = call @circuit(%arg0) : (tensor<f64>) -> tensor<f64>
```

Comment:
But line-to-long is disabled :(

Reply:
sorry, not sure I get what you mean. What's the issue here? I don't think codefactor is complaining here no?

Reply:
it was previously complaining!